com.unity.render-pipelines.high-definition/Documentation~/HDRP-Features.md

```diff
@@ -295,6 +309,18 @@ HDRP uses a physically-based Camera system that works seamlessly with the other
 
 Custom Passes allow you to inject shader and C# at certain points inside the render loop, giving you the ability to draw objects, do fullscreen passes and read some camera buffers like depth, color or normal, see the [Custom Pass documentation](Custom-Pass.md).
 
+### Custom Pass AOV Export
+
+![img](Images/aov_example.png)
+
+This feature allows you to export arbitrary data from custom pass injection points using an extension of the Arbitrary Output Variables (AOV) API in HDRP. An example use-case is for exporting “Object IDs” that are rendered with a custom pass. For information about the feature and example scripts, see the [AOV documentation](AOVs.md).
+
+### Custom Pass API
+
+The Custom Pass API allows you to render GameObjects from another point of view, like a disabled camera, within the rendering of your main Camera. This API also comes with built-in support for rendering Depth, Normal and Tangent into an RTHandle.
```

Comment:
this API shouldnt be a separate section but should be inside ### Custom Passes section following the other description.